### ISSUE TITLE
Need help making a motor turn a certain amount of degress.

### ISSUE THREAD

**Issue by RMariegaard** [jaraco/jaraco.nxt#3]
Hey, I'm currently using the project for a university project, wherein I need a motor to turn 90 degrees. I can see from the documentation on similar projects that the tacho_limit might be used for this. I tried running the motor with a tacho limit, but it doesn't seem to correspond to degrees. furthermore, the amount of time the motor runs for seems to vary, even though I don't change the tacho limit. Any help is much appreciated.

**Comment by jaraco** [comment]
Hi R. Sorry for the delay. I only today saw your message.

I haven't worked with this project in a long time and I don't recall if what you seek is possible.

Did you ever find a solution?

**Comment by jaraco** [comment]
Maybe you need to set a regulation mode to 'motor_speed' or 'motor_sync'.

**Comment by RMariegaard** [comment]
Hey Jason, we ended up using a Lego Mindstorms EV3 and micro python instead of the nxt for our project. Sadly with the EV3 we never could get the bluetooth connectivity to work quite as well as we did using your library and the NXT. Even more sadly it seems like all the most popular libraries for these old devices are deprecated, the times they are a-changing :(. Many thanks for the response from a group of former university students anyhow!
